gui/widgets/matRad_DVHStatsWidget.m

```diff
@@ -1,7 +1,7 @@
 classdef matRad_DVHStatsWidget < matRad_Widget
```

Comment:
The stats widget has the same issue as the DVH widget. There exist's a "qi" struct from the elemental calculation that could be cached / set directly for display.

Reply:
I do not understand this.